.github/workflows/ci.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     steps:
     - name: Checkout code
       uses: actions/checkout@v4
```

Comment:
What does `fetch-depth: 0` do?  Does it stop some kind of recursive fetching if you have stashed versions of the code?  Just curious

Reply:
`fetch-depth: 0` tells the checkout action to get all of the tags (and everything).  This is needed by `versioningit` to determine the version.